server/plugin/approve.go

```diff
@@ -25,9 +25,27 @@ func (p *Plugin) httpHandleApprove(w http.ResponseWriter, r *http.Request) {
 	originalPost, appErr := p.API.GetPost(requestData.PostId)
 	if appErr != nil {
 		p.API.LogError("Unable to get post", "postID", requestData.PostId)
-	} else if _, appErr := p.API.UpdatePost(originalPost); appErr != nil {
-		// TODO : remove the button
-		p.API.LogError("Unable to update post", "postID", originalPost.Id)
+	} else {
+		newAttachments := []*model.SlackAttachment{}
+		for _, attach := range originalPost.Attachments() {
+			filteredAttach := attach
+			attach.Actions = nil
+			for _, action := range attach.Actions {
+				if action.Id != "approvecirclecijob" {
+					filteredAttach.Actions = append(filteredAttach.Actions, action)
+				}
+			}
+
+			filteredAttach.Color = "#50F100" // green
+			filteredAttach.Title = "This CircleCI workflow have been approved"
```

Comment:
Is it possible to add who approved?

Reply:
![image](https://user-images.githubusercontent.com/45119518/95131589-9b5fd000-075e-11eb-8aed-99381cd091b4.png)
Of course! Just pushed :-)